pkg/gatherers/clusterconfig/machine_configs_test.go

```diff
@@ -32,7 +32,7 @@ func Test_SAPMachineConfigs(t *testing.T) {
 		machineConfigGroupVersionResource: "MachineConfigsList",
 	})
 
-	records, errs := gatherSAPMachineConfigs(context.Background(), machineConfigClient)
+	records, errs := gatherMachineConfigs(context.Background(), machineConfigClient)
```

Comment:
Please also change the name of the test function to match the gatherer function. It confused me a lot when running the tests because I thought I was still on some kind of an old branch or running into caching issues.

Reply:
Good catch. Thanks